android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -377,18 +487,37 @@ public void stopStreaming() {
         stateMachine.transitionToState(StreamingStateMachine.STOPPED);
     }
 
+    /**
+     * Stops streaming from the remote display. To restart, call
+     *
+     * @see #resumeStreaming()
+     */
+    private void stopStreaming(boolean withPendingRestart) {
+        if (remoteDisplay != null && !withPendingRestart) {
+            remoteDisplay.stop();
+            this.withPendingRestart = false;
+        }
+        if (this.isStreaming()) {
+            if (virtualDisplayEncoder != null) {
+                virtualDisplayEncoder.shutDown();
+            }
+            stateMachine.transitionToState(StreamingStateMachine.STOPPED);
+            stopVideoStream();
+        }
+    }
+
     /**
      * Resumes streaming after calling
      *
      * @see #startRemoteDisplayStream(android.content.Context, Class, com.smartdevicelink.streaming.video.VideoStreamingParameters, boolean)
      * followed by a call to
-     * @see #stopStreaming()
+     * @see #stopStreaming(boolean withPendingRestart)
```

Comment:
This should refer back to `stopStreaming()` instead of `stopStreaming(boolean withPendingRestart)`